### PR TITLE
CSS, Small Bugs and Refactoring

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: web/admin/src/Admin.js [hunk 1]
```diff
@@ -196,27 +194,21 @@ export default class Admin extends Component {
       })
 
       modRef.on('child_added', data => {
-        this.setState({ moderator: [...this.state.moderator, { ...data.val(), key: data.key }] })
+        this.setState({ moderator: { ...data.val(), key: data.key } })
```
Comment (AdamLiechty) [inline web/admin/src/Admin.js]: I don't think I understand this change.  Each `child_added` will replace the state's `moderator`?

File: web/admin/src/containerHeader.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ * limitations under the License.
+ */
+
+import React, { memo } from 'react'
+import { translate as t } from '@doubledutch/admin-client'
+import Select from 'react-select'
+import ModIcon from './modicon'
+import PresentationDriver from './PresentationDriver'
+
+const ContainerHeader = props => {
```
Comment (AdamLiechty) [inline web/admin/src/containerHeader.js]: It's customary to name the file the same as the React component (same capital casing)

File: web/admin/src/editCell.js [hunk 3]
```diff
@@ -53,6 +53,7 @@ export class CellEdit extends Component {
             className="ddLink"
             id={task.key}
             type="text"
+            onChange={e => console.log('edit')}
```
Comment (AdamLiechty) [inline web/admin/src/editCell.js]: remove?

File: web/admin/src/tableHeader.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { memo } from 'react'
+import { translate as t } from '@doubledutch/admin-client'
+
+const TableHeader = props => {
```
Comment (AdamLiechty) [inline web/admin/src/tableHeader.js]: you can destructure these props in the parameter, like
```javascript
const TableHeader = ({
  questions,
  status,
  modOff,
  handleApproved,
  handleApprovedPop,
  handleClick,
  handleAnswer,
}) => {
  ...
}
```

File: web/admin/src/Admin.js (hunk 1 above)
Comment (edwardstamps) [reply web/admin/src/Admin.js]: Changing this to moderation as moderator is a bit confusing on its purpose as its only 1 value

File: web/admin/src/editCell.js (hunk 3 above)
Comment (edwardstamps) [reply web/admin/src/editCell.js]: I was getting a warning for a textInput without any onChange is the reasoning here

Comment (AdamLiechty) [reply web/admin/src/editCell.js]: Hm... this warning is probably meant to steer you in a different direction.  Is this input meant to be edited?  If not, it should probably be disabled (with no onChange).

File: web/admin/src/settingsContainer.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { memo } from 'react'
+import { translate as t } from '@doubledutch/admin-client'
+import AnomIcon from './anomicon'
+
+const SettingsContainer = props => {
+  const {
```
Comment (AdamLiechty) [inline web/admin/src/settingsContainer.js]: Same inlining of props as TableHeader

**Pull request merged**